src/dotnet/Program.cs

```diff
@@ -129,13 +139,7 @@ internal static int ProcessArgs(string[] args, ITelemetry telemetryClient)
                 exitCode = result.ExitCode;
             }
 
-            telemetryClient.TrackEvent(
-                command,
-                null,
-                new Dictionary<string, double>
-                {
-                    ["ExitCode"] = exitCode
```

Comment:
I'd rather drop exit code than incur the delay at the end. Post-RC2 we could look at re-introducing it by having it shell out a new process that won't block the foreground process to capture the exit code and other data.